InAppSettingsKitFramework/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.7.2</string>
 	<key>CFBundleSignature</key>
```

Comment:
Hm, unless we dynamically set this (i.e. from the .podspec file), I foresee we'll forget to update the version number here.